canonicalwebteam/blog/django/views.py

```diff
@@ -15,16 +15,39 @@
 
 
 def index(request):
-    page_param = request.GET.get("page", default=1)
+    page_param = request.GET.get("page", default="1")
 
     try:
-        articles, total_pages = api.get_articles(
-            tags=tag_ids, tags_exclude=excluded_tags, page=page_param
-        )
+        if page_param == "1":
+            featured_articles, total_pages = api.get_articles(
+                tags=tag_ids,
+                tags_exclude=excluded_tags,
+                page=page_param,
+                sticky="true",
```

Comment:
I assume this is what makes them "featured"?

Reply:
Yes, exactly.